### PR TITLE
KAH-26_built-configure-game-screen

### PR DESCRIPTION
**Pull request by WojciechKaluzinski**

### CODE REVIEW

File: lib/features/game_config/views/game_config_page.dart [hunk 1]
```diff
@@ -14,16 +19,50 @@ class GameConfigPage extends StatelessWidget {
...
+    gameNameInputController.addListener(_setCurrentPlayerName);
+  }
+
+  @override
+  void dispose() {
+    gameNameInputController.dispose();
+    super.dispose();
+  }
+
+  _setCurrentPlayerName() {
```
Comment (joachimschmidt) [inline lib/features/game_config/views/game_config_page.dart]: _setCurrentGameName ?

File: lib/features/game_config/views/game_config_page.dart [hunk 2]
```diff
@@ -33,21 +72,102 @@ class GameConfigPageView extends StatelessWidget {
...
+                          items: pageState.availableCategories.map((category) {
+                            return DropdownMenuItem(
+                              child: Text(category.name),
+                              value: category,
+                            );
+                          }).toList(),
+                        ),
+                      ),
+                      SizedBox(height: 15),
+                      Padding(
```
Comment (joachimschmidt) [inline lib/features/game_config/views/game_config_page.dart]: Every _ConfigureNumberPicker has this pading. Including it in the _ConfigureNumberPicker will be a better idea

The best solution tho would be wrapping entire column in horizontal padding, becouse every widget in it( apart from Text, but who cares) is wrapped in horizontal padding 30

And you don't need vertical padding when you have SizedBoxes! Which are great for creating space in columns and rows.

So do this:
Wrap Column in padding symmetric horizontal 30 and increse heights of sized boxes so they match padding vertical spacing. And remove these separate paddings

File: lib/features/game_config/views/game_config_page.dart [hunk 3]
```diff
@@ -58,3 +178,73 @@ class GameConfigPageView extends StatelessWidget {
     );
   }
 }
+
+class _ConfigureNumberPicker extends StatefulWidget {
```
Comment (joachimschmidt) [inline lib/features/game_config/views/game_config_page.dart]: _GameConfigNumberPicker (configure number picker sounds like you're using it to configure number picker

File: pubspec.yaml [hunk 4]
```diff
@@ -18,6 +18,7 @@ dependencies:
   dartz: ^0.10.0-nullsafety.1
   kahootify_server:
     path: packages/server
+  numberpicker: ^2.1.1
```
Comment (joachimschmidt) [inline pubspec.yaml]: move it above kahootify_server. That's remote package and it's place is among it's kind

**Review by joachimschmidt — CHANGES_REQUESTED** [review]
Superb progress so far Wojciech! Keep it going! I have some sugestions for you tho :) Take a look and let me know if you will need any asistance in implementig those ;)

File: lib/features/game_config/views/game_config_page.dart (hunk 1 above)
Comment (WojciechKaluzinski) [reply lib/features/game_config/views/game_config_page.dart]: Ready

File: lib/features/game_config/views/game_config_page.dart (hunk 3 above)
Comment (WojciechKaluzinski) [reply lib/features/game_config/views/game_config_page.dart]: Ready

File: lib/features/game_config/views/game_config_page.dart (hunk 2 above)
Comment (WojciechKaluzinski) [reply lib/features/game_config/views/game_config_page.dart]: Ready

File: pubspec.yaml (hunk 4 above)
Comment (WojciechKaluzinski) [reply pubspec.yaml]: ready

**Review by joachimschmidt — APPROVED** [review]

**Pull request merged**